src/applications/personalization/profile/components/contact-information/ContactInformationEditView.jsx

```diff
@@ -306,6 +287,15 @@ export class ContactInformationEditView extends Component {
               }
               onSubmit={onSubmit}
             >
+              {error && (
+                <div
+                  role="alert"
+                  className="vads-u-margin-bottom--2 vads-u-margin-top--4"
```

Comment:
Heads up: these margins on the alert _might_ result in there being _too much_ space between the bottom of the alert and the top of the submit/cancel buttons once the "missing button padding" issue (shown in the screenshot) is resolved.

<img width="554" alt="Screen Shot 2021-11-29 at 7 44 16 AM" src="https://user-images.githubusercontent.com/20728956/144254486-9cad1d95-57ae-4cc6-bcf9-f70146b3a454.png">

Reply:
Oh good call. I'll remove the margin bottom then.